src/Lykke.Job.CandlesHistoryWriter/Models/Migration/TradesMigrationRequestModel.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace Lykke.Job.CandlesHistoryWriter.Models.Migration
+{
+    /// <summary>
+    /// API request model for trades migration routine.
+    /// </summary>
+    public class TradesMigrationRequestModel
+    {
+        /// <summary>
+        /// The date (and time) wich limits the trades selection (and old candles replacing) from the top. Please, use UTC time zone.
+        /// </summary>
+        public DateTime? TimestampUpperLimit { get; set; }
+
+        /// <summary>
+        /// An array of strings representing asset pair IDs selected for trades migration in the current session.
+        /// </summary>
+        [Required]
+        public string[] AssetPairIds { get; set; }
+
+        public bool CheckupModel(out Dictionary<string, List<string>> modelErrors)
```

Comment:
You can use MVC built-in class ModelStateDictionary. You do not need mark it as out parameter. Just pass ModelState to this method, fill it up and do ModelState.IsValid in controller. This will be more conventional